doc/nrf/links.txt

```diff
@@ -152,6 +152,7 @@
 
 .. _`Network Traffic Generator`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/connectivity/networking/api/zperf.html#zperf-network-traffic-generator
 
+.. _`commit #63e889`: https://github.com/nrfconnect/sdk-nrf/pull/9561/commits/63e889a179577ef08161923830bf2f87c8733fea
```

Comment:
I think we should refer only to the actual commits hashes in sdk-nrf repository (the hash will survive as long as repository itself is available, i.e. even if hosting provider changes the user can manually extract the hash and check it using git) and not to the PR versions of the commits (which is tied to github and to the fork from which the PR is opened). Here it would be ``https://github.com/nrfconnect/sdk-nrf/commit/d553142f4f37bc324da9869e4837cdd9105e25ab``

Reply:
true, I will correct